tabels.sql

```diff
@@ -1,13 +1,20 @@
 -- Generated from: https://drawsql.app/teams/get-shit-done/diagrams/oribookkeeping
 -- How to connect: PGPASSWORD='example' psql -U postgres -h localhost -d mydatabase
 
-CREATE TABLE "accounts"(
-    "id" BIGINT NOT NULL,
-    "owner_id" UUID NOT NULL
+CREATE TABLE IF NOT EXISTS "accounts"(
+    "id" UUID NOT NULL,
+    "owner_id" UUID NOT NULL,
+    "account_number" TEXT NOT NULL,
+    "type" TEXT NOT NULL,
+    "balance_cents" BIGINT NOT NULL,

```

Comment:
Alternatively, instead of putting a zero value when you do the INSERT request, you can do something like:
```
"balance_cents" BIGINT NOT NULL DEFAULT(0)
```